frontend/model/state.js

```diff
@@ -515,7 +515,11 @@ const store: any = new Vuex.Store({
 
 // save the state each time it's modified, but debounce it to avoid saving too frequently
 const debouncedSave = _.debounce(() => sbp('state/vuex/save'), 500)
-store.subscribe(debouncedSave) // for e.g saving notifications that are markedAsRead
+store.subscribe((commit) => {
+  if (commit.type !== 'noop') {
```

Comment:
(Pls correctly me if I'm wrong)
['noop'](https://github.com/okTurtles/group-income/blob/master/frontend/model/state.js#L132) commit triggered [here](https://github.com/okTurtles/group-income/blob/master/frontend/model/state.js#L529) in 'EVENT_HANDLED' handler apparently is some sort of a dummy(?) action created to help with the possible discrepancy in Vuex devtool state.

as this commit doesn't actually perform any state change(it's an empty function), the state doesn't need to be cached in the local-storage in response to it.